packages/core/src/Account.ts

```diff
@@ -406,30 +406,30 @@ export class Account extends TypedEventEmitter<Events> {
   }
 
   private async buildCryptoClient(context: Context, storeEngine: CRUDEngine) {
-    /* There are 3 cases where we want to instantiate CoreCrypto:
-     * 1. MLS is enabled
-     * 2. The user has enabled CoreCrypto in the config
-     * 3. The user has already used CoreCrypto in the past (cannot rollback to using cryptobox)
-     */
-    const clientType =
-      (await this.isMlsEnabled()) ||
-      !!this.cryptoProtocolConfig?.useCoreCrypto ||
-      cryptoMigrationStore.coreCrypto.isReady(storeEngine.storeName)
-        ? CryptoClientType.CORE_CRYPTO
-        : CryptoClientType.CRYPTOBOX;
```

Comment:
This could be simplified, basically we init `CoreCrypto` as soon as this object is defined (no other conditions)